src/uniprotkb/components/entry/tabs/genomic-coordinates/Coordinates.tsx

```diff
@@ -0,0 +1,178 @@
+
+  if (!taxID || !ensemblInfo?.uriLink) {
+    return null;
+  }
+
+  const infoData = [
+    {
+      title: 'Ensembl Gene',
+      content: coordinates.ensemblGeneId && (
+        <ExternalLink
+          url={processUrlTemplate(ensemblInfo.uriLink, {
+            id: coordinates.ensemblGeneId,
+          })}
+        >
+          {coordinates.ensemblGeneId}
+        </ExternalLink>
+      ),
+    },
+    {
+      title: 'Ensembl Transcript',
+      content: coordinates.ensemblTranscriptId && (
+        <ExternalLink
+          url={processUrlTemplate(ensemblInfo.uriLink, {
+            id: coordinates.ensemblTranscriptId,
+          })}
+        >
+          {coordinates.ensemblTranscriptId}
+        </ExternalLink>
+      ),
+    },
+    {
+      title: 'Ensembl Translation',
+      content: coordinates.ensemblTranslationId && (
+        <ExternalLink
+          url={processUrlTemplate(ensemblInfo.uriLink, {
+            id: coordinates.ensemblTranslationId,
+          })}
+        >
+          {coordinates.ensemblTranslationId}
+        </ExternalLink>
+      ),
+    },
+    {
+      title: 'Genomic location',
+      content: gl.start && gl.end && (
+        <GenomicLoc genomicLocation={gl} taxID={taxID} />
+      ),
+    },
+    {
+      title: 'Strand',
+      content: gl.reverseStrand ? 'Reverse' : 'Forward',
+    },
+    {
+      title: 'Number of exons',
+      content: gl.exon.length,
+    },
+    {
+      title: 'Overlapping proteins',
+      content: gl.start && gl.end && (
+        <LazyComponent
+          fallback={<SpinnerIcon width="1em" height="1em" />}
+          rootMargin="0px"
+        >
+          <Overlapping
+            taxID={taxID}
+            chromosome={gl.chromosome}
+            start={gl.reverseStrand ? gl.end : gl.start}
+            end={gl.reverseStrand ? gl.start : gl.end}
+            currentEntry={currentEntry}
+          />
+        </LazyComponent>
+      ),
+    },
+  ];
+
+  return (
+    <section>
+      <h3>Genomic location {index + 1}</h3>
+      <InfoList infoData={infoData} columns />
+      <DatatableWithToggle>
+        <table>
+          <thead>
+            <tr>
+              <th>Exon ID</th>
+              <th>Protein coordinates</th>
+              <th>Genomic coordinates</th>
+            </tr>
+          </thead>
+          <tbody>
+            {coordinates.genomicLocation.exon.map((exon) => (
+              <tr key={exon.id}>
+                <td>
+                  {exon.id && ensemblInfo.uriLink && (
+                    <ExternalLink
+                      url={processUrlTemplate(ensemblInfo.uriLink, {
+                        id: exon.id,
+                      })}
+                    >
+                      {exon.id}
+                    </ExternalLink>
+                  )}
+                </td>
+                <td>
+                  <LongNumber>
+                    {exon.proteinLocation?.begin?.position ?? ''}
+                  </LongNumber>
+                  -
```

Comment:
Noticed it's a bit difficult to read as their are so many numbers in this table so thought a bit of space could help eg
```suggestion
                  {' - '}
```

Reply:
covered in 52b9fdeabfa507029e73e580e51982ecae7f595a